includes/definitions/lcos.yaml

```diff
@@ -6,7 +6,9 @@ over:
     - { graph: device_bits, text: 'Device Traffic' }
     - { graph: device_processor, text: 'CPU Usage' }
     - { graph: device_mempool, text: 'Memory Usage' }
+register_mibs:
+    LCOS-MIB: LCOS-MIB
```

Comment:
I think this is wrong, I assume you just want to tell things where the mib is?

You should do:

```
mib_dir:
    - lancom

```